cumulusci.yml

```diff
@@ -250,6 +261,7 @@ plans:
                         name: K-12 Architecture Kit Metadata and Configuration
             5:
                 task: deploy_k12_app
+                
```

Comment:
at some point I can see us adding it to our dev scratch orgs as well. For now, probably not a big deal.